### DISCUSSION TITLE
Ps4 not pairing to Nintendo switch 12.1 - 19.5

### DISCUSSION THREAD

**Discussion by Endongshi** [ndeadly/MissionControl#352] [Q&A]
I can't pair my DS4 to my nintendo switch. I downloaded the latest version and restarted my switch and still I can't pair it. Below is the content of the file I downloaded 

![Screenshot_2021-09-20-13-52-33-676_com mi android globalFileexplorer](https://user-images.githubusercontent.com/91043709/133961942-89591fe4-c89e-4fa3-9c72-5c791433062c.jpg)

**Comment by ndeadly** [comment]
Please don't create multiple posts on the same issue.

If you've restarted your switch already and the controller is still not pairing I see three likely possibilities.

Either,
1) you haven't followed the [usage](https://github.com/ndeadly/MissionControl#usage) or [pairing](https://github.com/ndeadly/MissionControl#pairing-controllers) instructions correctly.

2) your controller is being stubborn to to pair to the switch. Some controllers, especially V1 DS4s can take multiple tries before they will connect. Pressing the reset button on the back can help, but otherwise you just need to keep trying until it pairs.

3) your controller is not an authentic Sony controller and may have an incompatible Bluetooth chipset.

Make sure you're following the instructions in 1) correctly, but I would guess 2) is likely your problem.

Also, I don't think the contents directory is required in your case as your issue is likely one of those mentioned above, but you're supposed to post the `atmosphere/contents` directory from your SD card, not the mission control archive you downloaded. The idea being to show what other sysmodules you may have installed.

**Reply by Endongshi** [reply]
Apologies sir. Was not able to delete my initial post. Upon checking the content of my SD card atmosphere/content. The same content as above. My DS4 is V2. will try again and read the "usage" and "pairing" and see if I can make this work.

**Reply by Endongshi** [reply]
I think this case can be close now. Upon reading how to check fake DS4. Mine fits all the criteria. The reason why I can't pair it on my nintendo switch.

**Reply by ndeadly** [reply]
A bootleg controller is not a guarantee it won't work, so it may still be worth trying to reset and pair several times like I mentioned above if you haven't already. But if that's not working for you and you've followed the pairing instructions correctly then I would assume it's a Bluetooth compatibility issue.

**Reply by Endongshi** [reply]
i tried pairing it also using an 8bitdo adapter and still it won't pair. will try again later
